src/sign/cosmos.tf

```
@@ -0,0 +1,30 @@
+module "cosmosdb_account" {
+  source                           = "git::https://github.com/pagopa/azurerm.git//cosmosdb_account?ref=v2.13.1"
+  name                             = "${local.project}-cosmos"
+  subnet_id                        = module.io_sign_snet.id
+  location                         = azurerm_resource_group.backend_rg.location
+  resource_group_name              = azurerm_resource_group.backend_rg.name
+  kind                             = "GlobalDocumentDB"
+  main_geo_location_location       = azurerm_resource_group.backend_rg.location
+  main_geo_location_zone_redundant = false
+  tags                             = var.tags
+}
+
+module "cosmosdb_sql_database_io_sign" {
+  source              = "git::https://github.com/pagopa/azurerm.git//cosmosdb_sql_database?ref=v2.13.1"
+  name                = "io-sign"
```

since we have a dedicated cosmos db we can remane the db "db"
```suggestion
  name                = "db"
```